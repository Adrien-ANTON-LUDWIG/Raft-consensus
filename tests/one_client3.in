--
LOAD file1.txt
LOAD file2.txt
LOAD file3.txt
LOAD file4.txt
LOAD file5.txt
LOAD file6.txt
$
start 3
@10000
exit