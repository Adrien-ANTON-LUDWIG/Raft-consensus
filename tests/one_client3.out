{"filename":"file1.txt","origin":3,"type":12}
{"filename":"file2.txt","origin":3,"type":12}
{"filename":"file3.txt","origin":3,"type":12}
{"filename":"file4.txt","origin":3,"type":12}
{"filename":"file5.txt","origin":3,"type":12}
{"filename":"file6.txt","origin":3,"type":12}